private/model/api/example.go

```diff
@@ -14,7 +14,41 @@ import (
 	"github.com/aws/aws-sdk-go-v2/private/util"
 )
 
-type Examples map[string][]Example
+// ExamplesGoCode will return a code representation of the entry within the
+// examples.json file.
+func (a *API) ExamplesGoCode() string {
+	a.resetImports()
+	a.AddImport("context")
+	a.AddImport("fmt")
+
+	a.AddSDKImport("aws")
+	a.AddSDKImport("aws/awserr")
+	a.AddSDKImport("aws/external")
+	a.AddSDKClientImport()
```

Comment:
We could have used `a.AddImport(a.ImportPath())` here instead of a.AddSDKClientImport()

This change looks good, but feels like we have lot of ways to Import and it may get confusing and result in different usages in the SDK for similar outcome.